scripts/_common.sh

```diff
@@ -1,35 +1,122 @@
 #!/bin/bash
-#
-# Common variables
-#
+
+#=================================================
+# COMMON VARIABLES
+#=================================================
 
 pkg_dependencies="php5-gd php5-imagick imagemagick"
```

Comment:
```suggestion
pkg_dependencies="php-gd php-imagick imagemagick"
```
Shouldn't we move to PHP7 dependencies...? :thinking: